Sources/QuickTests/Helpers/QuickSpec+QuickSpec_MethodList.h

```diff
@@ -0,0 +1,7 @@
+#import <Quick/Quick.h>
+
+@interface QuickSpec (QuickSpec_MethodList)
+
++ (NSSet<NSString*> *)allSelectors;
+
+@end
```

Comment:
Wrap this header in `NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END`, to improve its bridging to Swift.